packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts

```diff
@@ -255,9 +254,11 @@ export default function setAlertText(alert: any, callback: any) {
             loadMarketsInfoIfNotLoaded([alert.to], () => {
               const marketInfo = selectMarket(alert.to);
               const outcome = calculatePayoutNumeratorsValue(
-                marketInfo,
-                alert.params._payoutNumerators,
-                alert.params._invalid
+                marketInfo.maxPrice,
```

Comment:
These are display prices. Is that correct for calculatePayoutNumeratorsValue method?

Reply:
In the original function in augur-ui, the first line was: ```  const { maxPrice, minPrice, numTicks, marketType } = market;```
so I think it should be ok to pass these in.

Reply:
Not a question about passing them. Its a question of are they display values or on chain values. 
Since both the sdk and ui are calling it. I wanted to confirm that display values ate passed in from the sdk. Ui already uses display prices.

Reply:
Good point, the getter is passing in the value from the log, so it's not using the display price. 
 Should we make it accept on-chain values instead?

Reply:
That is the challenge, prob need to rename the method params to indicate if onChain or display so we dont make mistakes on these type of helper methods. UI only has display values. Could do this:

Have two different methods one for onchain and the other for display 
Or just take in display values. What ever you think fix best in the sdk.

Reply:
Actually, seems easier to keep it as display price since theres a conversion function from onchain to display in the middleware.

I'll rename the input params to be clearer.

Reply:
That makes the most sense to me.